randovania/gui/ui_files/preset_generation.ui

```diff
@@ -119,6 +119,53 @@
              </property>
             </widget>
            </item>
+           <item>
+            <widget class="Line" name="logical_pickup_placement_line">
+             <property name="orientation">
+              <enum>Qt::Orientation::Horizontal</enum>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <layout class="QHBoxLayout" name="logical_pickup_placement_layout">
+             <item>
+              <widget class="QLabel" name="logical_pickup_placement_label">
+               <property name="text">
+                <string>[Experimental] Logical pickup placement:</string>
+               </property>
+              </widget>
+             </item>
+             <item>
+              <widget class="QComboBox" name="logical_pickup_placement_combo">
+               <item>
+                <property name="text">
+                 <string>Minimal</string>
+                </property>
+               </item>
+               <item>
+                <property name="text">
+                 <string>Major pickups</string>
+                </property>
+               </item>
+               <item>
+                <property name="text">
+                 <string>All pickups</string>
+                </property>
+               </item>
+              </widget>
+             </item>
+            </layout>
+           </item>
+           <item>
+            <widget class="QLabel" name="logical_pickup_placement_description">
+             <property name="text">
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This setting controls what pickups Randovania places logically. Any pickups placed in this way are guaranteed to be obtainable. Other pickups may not be required to finish or even be possible to get.&lt;/p&gt;&lt;p&gt;Minimal: Only the pickups required to beat the game are placed logically. This is the default setting.&lt;/p&gt;&lt;p&gt;Major pickups: Every major pickup in the preset is placed logically.&lt;/p&gt;&lt;p&gt;All pickups: Every pickup, including ammo and expansions, is placed logically.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Warning:&lt;/span&gt; Selecting 'Major pickups' or 'All pickups' will make generation fail more frequently.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Not fully happy with the description here. Mostly the first paragraph before the minimal explanation.
Just having:
"This setting controls what pickups are guaranteed to be obtainable"
And then changing "placed logically" to "guaranteed to be obtainable" in the individual description sounds nicer to me.